### ISSUE TITLE
Training CNN: Dropout Layer effect

### ISSUE THREAD

**Issue by Goldesel23** [vlfeat/matconvnet#648]
Hi to all, 

So I was trying to explore different architectures and fine-tuning of pre-trained models, and I realized that the use of a dropout layer (tested with different rates) led to a non-decreasing loss function, while if that layer was not added the CNN would have a good behavior in the loss function. 

I tried this with relatively small CNN's (3/4 conv layers). Does anyone have a suggestion on why this might occur ?
